Curtains/Curtains.Infrastructure/Repositories/AccessoriesRepository.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Curtains.Domain.Models;
+using Curtains.Infrastructure.Database;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Infrastructure.Repositories
+{
+    public class AccessoriesRepository
+    {
+        private readonly ILogger _logger;
+        private readonly CurtainsDbContext _curtainsContext;
+        private DbSet<AccessoriesModel> accessories;
+
+        public AccessoriesRepository(CurtainsDbContext curtainsContext, ILogger logger)
+        {
+            _curtainsContext = curtainsContext;
+            _logger = logger;
+            accessories = _curtainsContext.Accessories;
+        }
+
+        public IEnumerable<AccessoriesModel> GetAll()
+        {
+            return accessories.AsEnumerable();
+        }
+
+        public async Task<AccessoriesModel> GetById(int Id)
+        {
+            return await accessories.SingleAsync(c => c.Id == Id);
```

Comment:
1 - you should add null check for it.
2 - Why you use Single instead of First, FirsOrDefault, SingleOrDefault ?